app/views/devise/invitations/edit.html.erb

```diff
@@ -31,7 +31,7 @@
     <div class="registration__field">
       <%= f.form_row :password_confirmation do %>
         <%= f.errors_for :password_confirmation %>
-        <%= f.label :password_confirmation, class: "form__label-heading" %>
+        <%= f.label :password_confirmation, t('authentication.accept_invitation_page.field_labels.password_confirmation'), class: "form__label-heading" %>
```

Comment:
Or are you referring to the fact that I'm not following the standard naming convention for views (which is a very good point and not sure how I missed that).